src/Application/Migrations/24_function_ticket_reset.sql

```diff
@@ -0,0 +1,70 @@
+BEGIN;
+
+--param_handle_id is used for the log entry
+CREATE OR REPLACE FUNCTION ticket_reset(param_ticket_id bigint, param_handle_id bigint)
+  RETURNS void AS
+$$
+DECLARE
+  ticket tbl_ticket%rowtype;
+  t tbl_ticket%rowtype;
```

Comment:
It looks like you could use the "ticket" variable in all cases, so you shouldn't need another variable.